SoftLayer/CLI/firewall/detail.py

```diff
@@ -12,31 +12,70 @@
 
 @click.command(cls=SoftLayer.CLI.command.SLCommand, )
 @click.argument('identifier')
+@click.option('--credentials', type=click.BOOL,
+              help="Display FortiGate username and FortiGate password to multi vlans.")
 @environment.pass_env
-def cli(env, identifier):
+def cli(env, identifier, credentials):
     """Detail firewall."""
 
     mgr = SoftLayer.FirewallManager(env.client)
 
     firewall_type, firewall_id = firewall.parse_id(identifier)
-    _firewall = mgr.get_instance(firewall_id)
 
-    table = formatting.KeyValueTable(['name', 'value'])
-    table.align['name'] = 'r'
-    table.align['value'] = 'l'
+    if firewall_type == 'vs' or firewall_type == 'server' or firewall_type == 'vlan' or firewall_type == 'multiVlan':
```

Comment:
This is a little easier to work with.
```suggestion
    if firewall_type in ('vs', 'server', 'vlan', 'multiVlan'):
```

Reply:
The conditional was updated following the suggestion